pkg/api/constants.go

```diff
@@ -1,6 +1,7 @@
 package api
 
 const (
-	APIEndpoint     = "https://api.opensauced.pizza/v1"
-	BetaAPIEndpoint = "https://beta.api.opensauced.pizza/v1"
+	AlphaAPIEndpoint = "https://alpha.api.opensauced.pizza/v1"
```

Comment:
Not often. We used it before when prototyping against different infrastructure, but haven't needed to use it since then. I don't think we need to include it